src/Console/Installer.php

```diff
@@ -99,7 +99,7 @@ public static function postInstall(Event $event)
     public static function createAppLocalConfig($dir, $io)
     {
         $appLocalConfig = $dir . '/config/app_local.php';
-        $defaultLocalConfig = $dir . '/config/app_local.default.php';
+        $defaultLocalConfig = $dir . '/config/app_local.example.php';
```

Comment:
If we really also make other side effects here now like renaming .default to .example (which IMO should be a follow up ticket/PR), then we must also adjust all other cases to be in sync here, e.g. https://github.com/cakephp/app/blob/master/config/.env.default

Reply:
Done?

Reply:
Example file is used as template. Variable should use _template_ keyword. Or both variables as _source_ and _destination_.